packages/icons/src/index.js

```diff
@@ -70,6 +70,7 @@ export { default as menu } from './library/menu';
 export { default as minus } from './library/minus';
 export { default as more } from './library/more';
 export { default as moreHorizontal } from './library/more-horizontal';
+export { default as moreVertifal } from './library/more-vertical';
```

Comment:
This is not used in the UI yet right?

Reply:
No, but when I get a chance I will. The top menu icon is vertical and is rotated with css, I'll remove the CSS and use the correct icon.

Reply:
Also, typo :)

Reply:
Fixed, that was embarrassing :D